.github/workflows/build-test.yml

```diff
@@ -0,0 +1,31 @@
+name: Build-Test
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+  push:
+    branches:
+      - 'master'
+
+jobs:
+
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone Repo
+        uses: actions/checkout@v1
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Run Unit tests
+        run: ./gradlew test --stacktrace
+
+      - name: Build project
```

Comment:
I think, that building of project must be before the tests and fail the whole job in case of failure in build

Reply:
Got it , I'll change